tests/parsers/agilent_gen5/agilent_gen5_structure_test.py

```diff
@@ -336,14 +337,37 @@ def test_create_layout_data_with_name_rows() -> None:
         "Layout",
         "\t1\t2\t3",
         "A\tSPL1\tSPL9\tSPL17\tWell ID",
-        "\tName_A1\tName_A2\tName_A3\tName",
```

Comment:
why did this go away?

Reply:
Oh yeah should have left a comment. So the logic here is that the `Name` column overrides the `Well ID` comment for identifiers, if present. I'm doing a little more magic now, so I'm testing here that it works on a per-column basis. Note that "A2" below now uses the Well ID because the name is missing for that column